### DISCUSSION TITLE
Web Authenticator - Close tab after connexion

### DISCUSSION THREAD

**Discussion by nterreaux** [dotMorten/WinUIEx#125] [Q&A] [answered]
Hello, I'm now using WinUIEx for my OAuth 2 authentication. Everything works perfectly except for the fact that the user's default browser page doesn't close automatically. This isn't extremely annoying, but is there a solution for closing the browser after authentication?

I'm using the version 2.2.0 of WinUIEx, .NET 7 and I'm doing a .NET MAUI application for Windows, Android, macOS and IOS

Thank you in advance for your reply.

Nicolas

**Comment by dotMorten** [comment, marked as answer]
Sorry WinUIEx has no control over how the browser deals with launching and redirecting back to apps. You’ll see the same behavior for all other apps that uses the default browser for oauth sign in. For example slack

**Reply by nterreaux** [reply]
Thank you very much for your prompt reply. I'll just stick with this, it's already great that it works on Windows despite the issue :)
